src/app/features/todos/todos-related-to-entity/todos-related-to-entity.component.ts

```diff
@@ -53,26 +51,28 @@ export class TodosRelatedToEntityComponent extends RelatedEntitiesComponent<Todo
     private dbIndexingService: DatabaseIndexingService,
     entityMapper: EntityMapperService,
     entities: EntityRegistry,
-    screenWidthOberserver: ScreenWidthObserver,
+    screenWidthObserver: ScreenWidthObserver,
+    filterService: FilterService,
   ) {
-    super(entityMapper, entities, screenWidthOberserver);
+    super(entityMapper, entities, screenWidthObserver, filterService);
+  }
+
+  override getData() {
+    if (Array.isArray(this.property)) {
+      return;
+    }
```

Comment:
when might this be an array - and is this a suitable fallback to not load then?

Reply:
It's an array if there is more than one property referencing the current entity. We currently do not support this because we don't have the required indices. We could however fallback to just using the entity mapper in this case.